style.css

```diff
@@ -1,3 +1,83 @@
-.color-blue {
-  color: blue;
+* {
+  font-family: "Poppins", sans-serif;
+  color: ash;
+}
+
+.main {
+  background-image: url("image-folder/header-shapes\ mobile@2x.svg");
+  background-color: #fff;
+  background-repeat: no-repeat;
+  background-size: cover;
+}
```

Comment:
- [OPTIONAL] KIndly make sure the background image covers the whole headline section and it has the border-radius on the left-bottom in order to match the provided Figma design.

   - ### YOUR DESIGN
   
     ![image](https://user-images.githubusercontent.com/87197412/227354124-7cfd6b56-d320-475d-9c2f-8e1183efbc21.png)

   - ### FIGMA DESIGN
   
     ![image](https://user-images.githubusercontent.com/87197412/227355112-b2826edd-2401-43fb-a64c-c899330b3e7c.png)